Snippets/Snippets_3/UnitTesting/Unobtrusive.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Snippets3.UnitTesting
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Testing;
+
+    public class Unobtrusive
+    {
+        public class Setup
+        {
+            public Setup()
+            {
+                #region SetupConventionsForUnitTests
+                MessageConventionExtensions.IsMessageTypeAction = MyConvention;
```

Comment:
The .md (that I'll add , missed that) is linking to the doco on onobtrusive. Is that enough?

Or perhaps `MyMessageTypeConvention` ?